ui/src/views/Hooks/ListHooks/index.jsx

```diff
@@ -89,7 +93,11 @@ export default class ListHooks extends Component {
             softSearch
             tree={tree}
             onEmptySearch={
-              <Typography variant="subtitle1">
+              <Typography
+                variant="subtitle1"
+                placeholder="Updated Hook contains"
```

Comment:
What does this mean?  I believe when this is displayed, there are no hooks, so nothing has been "updated"..